Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -63,6 +63,8 @@ protected static void SteamAPIDebugTextHook(int nSeverity, System.Text.StringBui
         HAuthTicket sessionTicketHandle = HAuthTicket.Invalid;
         string sessionTicketString = null;
 
+        Callback<GetTicketForWebApiResponse_t> m_AuthTicketForWebApiResponseCallback;
```

Comment:
Because this is a MonoBehaviour, I really want everything to have accessors. Otherwise Unity might do something funny and serialize the value of this, even though it is private. Do you think that applies here?

Reply:
Just learned a lot about serialization, after a crash course with Mint.

SteamManager only gets created a singleton-ish instance in runtime, and there are no other references in scenes, so there should be nothing getting serialized here.
Also `Callback` here is an object from Steamworks, which Unity wouldn't be able to serialize.